portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,56 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random Bible verse to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let bibleVerses = new Map();
+bibleVerses.set('Romans 5: 3-5', 'Not only so, but we also glory in our ' +
+		'sufferings, because we know that suffering produces perseverance; ' +
+		'perseverance, character; and character, hope. And hope does not put us ' +
+		'to shame, because God’s love has been poured out into our hearts ' +
+		'through the Holy Spirit, who has been given to us.');
+bibleVerses.set('Romans 10:13', 'For, “Everyone who calls on the name of ' +
+		'the Lord will be saved.”');
+bibleVerses.set('Psalms 94:19', 'When anxiety was great within me, your ' +
+		'consolation brought me joy.');
+bibleVerses.set('John 3:16', 'For God so loved the world that he gave his ' +
+		'one and only Son, that whoever believes in him shall not perish but ' +
+		'have eternal life.');
+bibleVerses.set('James 1:5', 'If any of you lacks wisdom, you should ask ' +
+		'God, who gives generously to all without finding fault, and it will be ' +
+		'given to you.');
+bibleVerses.set('Joshua 1:9', 'Have I not commanded you? Be strong and ' +
+		'courageous. Do not be afraid; do not be discouraged, for the Lord your ' +
+		'God will be with you wherever you go.');
+
+function getRandomBibleVerse() {
+	// Pick random index
+  const bibleVerseIndex = Math.floor(Math.random() * bibleVerses.size);
+	
+	// Select index from map
+	let index = 0;
+	let text = '';
+	let verse = '';
+	for(let key of bibleVerses.keys()) {
+		if(index++ === bibleVerseIndex) {
+			text = bibleVerses.get(key);
+			verse = key;
+		}
+	}
+	const bibleVerse = [text, verse];
+	return bibleVerse;
+}
+
+function addRandomBibleVerse() {
+  // Pick a random Bible verse.
+	const bibleVerse = getRandomBibleVerse();
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const bibleVerseTextContainer = 
+			document.getElementById('bible-verse-text-container');
+  bibleVerseTextContainer.innerText = bibleVerse[0];
+  const bibleVerseContainer = 
+			document.getElementById('bible-verse-container');
```

Comment:
Line wrapping should be an indent of 4 spaces - https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping. https://google.github.io/styleguide/jsguide.html#formatting-function-arguments also gives you a good idea of what this would look like.

Reply:
Again, weird, I did check the documentation after the first time you mentioned it, and I added four spaces to each continuation line. Don't know why it appears to have more...